docker-compose.yml

```diff
@@ -108,17 +151,103 @@ services:
       - exec_data:/data
     depends_on:
       - redis
-    deploy:
```

Comment:
was this on purpose?

Reply:
Yes, only a single worker should run as 4 is a bit of an overhead.

Reply:
ok, I would leave it in but only with a single replica to keep in mind that it is possible defining multiple workers :D